addons/mrp/report/mrp_report_bom_structure.xml

```diff
@@ -1,110 +1,105 @@
-                                        <tr class="o_mrp_bom_cost">
-                                            <td/>
-                                            <t t-foreach="range(data.get('extra_column_count', 0))" t-as="index">
-                                                <td/>
-                                            </t>
-                                            <td name="td_mrp_bom_byproducts_f" class="text-end">
-                                                <span><t t-esc="byproduct['product_name']"/></span>
-                                            </td>
-                                            <td class="text-end"><span><strong>Unit Cost</strong></span></td>
-                                            <td groups="uom.group_uom"><span><t t-esc="byproduct['product_uom']"/></span></td>
-                                            <td class="text-end">
-                                                <span><t t-esc="byproduct['product_cost'] / byproduct['product_qty']" t-options='{"widget": "monetary", "display_currency": currency}'/></span>
-                                            </td>
-                                            <td class="text-end">
-                                                <span><t t-esc="byproduct['cost_share'] * data['total'] / byproduct['product_qty']" t-options='{"widget": "monetary", "display_currency": currency}'/></span>
-                                            </td>
-                                        </tr>
+                <table class="o_mrp_bom_expandable table">
+                    <thead>
+                        <tr>
+                            <th>Product</th>
+                            <th name="th_mrp_bom_h">BoM</th>
+                            <th class="text-end">Quantity</th>
+                            <th class="text-start" groups="uom.group_uom">Unit of Measure</th>
+                            <th t-if="data['report_structure'] != 'bom_structure'" class="o_mrp_prod_cost text-end" title="This is the cost defined on the product.">Product Cost</th>
+                            <th t-if="data['report_structure'] != 'bom_structure'" class="o_mrp_bom_cost text-end" title="This is the cost based on the BoM of the product. It is computed by summing the costs of the components and operations needed to build the product.">BoM Cost</th>
+                            <th t-if="data['report_type'] == 'html' and data['has_attachments']" class="o_mrp_has_attachments" title="Files attached to the product">Attachments</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>
+                                <span><a href="#" t-if="data['report_type'] == 'html'" t-att-data-res-id="data['product'].id" t-att-data-model="'product.product'" class="o_mrp_bom_action"><t t-esc="data['bom_prod_name']"/></a><t t-else="" t-esc="data['bom_prod_name']"/></span>
```

Comment:
Could we take the opportunity to get rid of all of these <span>? Seems not necessary...